ts/nni_manager/training_service/reusable/aml/amlClient.ts

```diff
@@ -135,11 +135,19 @@ export class AMLClient {
     
     // Monitor error information in aml python shell client
     private monitorError(pythonShellClient: PythonShell, deferred: Deferred<any>): void {
+        pythonShellClient.on('stderr', function (error: any) {
+            // FIXME: The error will only appear in console.
+            // Still need to find a way to put them into logs.
+            console.error(`Python process stderr: ${error}`);
```

Comment:
```typescript
import { getLogger } from 'common/log';
getLogger('AmlClient').error('Python process stderr:', error);
```
Prefer comma separated parameters over template string. The logger uses built-in `inspect()` util to keep as much info as possible, while the default string converting commonly generates meaningless `[Object object]`.